### ISSUE TITLE
More details on pretraining guidance would be helpful

### ISSUE THREAD

**Issue by awwkl** [facebookresearch/vjepa2#56]
V-JEPA's training loss is not grounded in reconstructing the real-world videos, as there is no decoder. This makes it challenging to train as the loss curves do not strictly go downwards over training, unlike a video next-frame prediction model. 

It would be super helpful if the authors of the repo release some guidance on the issues they faced during pretraining / cooldown / evaluation probe, what their training loss curves look like, and what checkpoints to select for each.

For example, I have this loss curve from pretraining the ViT-L version of V-JEPA 2 on Kinetics400 (x-axis is epoch) with default hyperparameters except for lower batch size due to lack of GPUs, but I am unsure if I should continue training this model, and which checkpoint to select to do the cooldown.

Your advice would be very helpful! Thank you.

<img width="640" height="480" alt="Image" src="https://github.com/user-attachments/assets/35064ef7-6eac-438e-89e7-76dc7f2817c3" />

**Comment by ajamal-isrg** [comment]
Have you guys tried evaluating the model at intermediate checkpoints?

**Comment by Jazzcharles** [comment]
I have observed similar trend while training a vit-large model on a subset of K400 and SSv2. It seems that the latest checkpoint (e.g. 400 ep) performs much better than early checkpoint (e.g. 100ep) on SSv2 attention probe, even though the latest checkpoint has a higher loss. 

I guess that is the reason the authors **check evaluations every 60k steps** as mentioned in their paper. 

<img width="1341" height="387" alt="Image" src="https://github.com/user-attachments/assets/8b37185a-61c1-41f9-b7e8-58337680937b" />

**Comment by dfan** [comment]
The pretraining loss is not a "traditional" loss in the sense that the target is moving (we use EMA). The loss curve you show is actually expected and matches what we see for VJEPA pretraining. There is probably potential to improve the training recipe to make the loss look more like a traditional loss, but downstream results (e.g. probe-based eval) are currently the best way to assess feature quality.
